platform/features/events/src/policies/MessagesViewPolicy.js

```diff
@@ -0,0 +1,66 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+/**
+ * Module defining MessagesViewPolicy. Created by shale on 06/24/2015.
+ */
+define(
+    [],
+    function () {
+        "use strict";
+
+        /**
+         * Policy controlling when the Messages view should be avaliable.
+         * @constructor
+         */
+        function MessagesViewPolicy() {
+            
+            return {
+                /**
+                 * Check whether or not a given action is allowed by this
+                 * policy.
+                 * @param {Action} action the action
+                 * @param domainObject the domain object which will be viewed
+                 * @returns {boolean} true if not disallowed
+                 */
+                allow: function (view, domainObject) {
+                    // This policy only applies for the Messages view
+                    if (view.key === 'messages') {
+                        // The Messages view is allowed only if the domain 
+                        // object is a Event Message Generator
+                        if (domainObject.getModel().type !== 'eventGenerator') {
+                            return false;
+                        }
```

Comment:
There's an example of this being done for imagery [here](https://github.com/nasa/openmctweb/blob/open1170/platform/features/imagery/src/policies/ImageryViewPolicy.js#L34-L44); for events, this should look similar, except the range format to look for will be `"string"` instead of `"image"` or `"imageUrl"`